client/my-sites/customer-home/main.jsx

```diff
@@ -420,11 +524,17 @@ class Home extends Component {
 }
 
 const connectHome = connect(
-	state => {
+	( state, props ) => {
 		const siteId = getSelectedSiteId( state );
 		const siteChecklist = getSiteChecklist( state, siteId );
 		const hasChecklistData = null !== siteChecklist && Array.isArray( siteChecklist.tasks );
 		const domains = getDomainsBySiteId( state, siteId );
+		let themeInfo = {};
+		if ( props.displayMode && 'theme' === props.displayMode ) {
```

Comment:
...if we assign `displayMode` above, we can just do:
```suggestion
		if ( 'theme' === displayMode ) {
```